### PR TITLE
Added new game Eco Warrior

### LINKED ISSUE #4296: [New game]: Eco Warrior
## Game logic and basic description
Eco Warrior is an educational and engaging game designed to raise awareness about environmental conservation. Players take on the role of an eco-warrior whose mission is to clean up litter, plant trees, and restore ecosystems. The game is designed to be both fun and educational, teaching players the importance of keeping the environment clean and green.

## How to Play

- **Objective**: Clean up all the litter within the given time to advance to the next level. Additionally, plant trees to increase your impact score and improve the environment.
- **Energy**: Start with 100 energy points. Cleaning litter and planting trees consume energy. Energy regenerates over time.
- **Score**: Earn points by cleaning litter and planting trees. Higher scores unlock upgrades and allow you to progress to higher levels.
- **Timer**: Each level has a countdown timer. Clean up all the litter before the time runs out to complete the level. The time available increases with each subsequent level.

## Game Elements

- **Litter**: Click on litter items scattered around the game area to clean them up. Each piece of litter cleaned increases your score.
- **Trees**: Plant trees to improve your score and the environment. Planting a tree costs 10 energy points.
- **Upgrades**: Accumulate 50 points to purchase an upgrade that increases your energy regeneration rate.

## Levels

The game consists of multiple levels, each progressively more challenging:
- **Level 1**: Start with 10 pieces of litter and 60 seconds to clean them up.
- **Subsequent Levels**: Each new level adds 5 more pieces of litter and increases the timer by 10 seconds. The challenge increases as you progress, requiring more strategic use of energy and time.

## Game Over

If you fail to clean up all the litter within the given time, the game will display a "Game Over" message and reset. Try again to beat your previous scores and advance further.

## Educational Impact

Eco Warrior aims to educate players about the importance of environmental conservation. By simulating the cleanup process and tree planting, players learn the value of these actions in a fun and interactive way. The game encourages players to think about real-world environmental issues and how they can contribute to a cleaner, greener planet.

### Select program in which you are contributing

GSSoC24

### Code of Conduct

- [X] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) of this project.

### PR DESCRIPTION
**Pull request by Antiquely3059**
## PR Description 📜
**Eco Warrior** is an educational and engaging game designed to raise awareness about environmental conservation. Players take on the role of an eco-warrior whose mission is to clean up litter, plant trees, and restore ecosystems. The game is designed to be both fun and educational, teaching players the importance of keeping the environment clean and green.

Fixes #4296 

<hr>
 
## Mark the task you have completed ✅

- [x] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) & [CODE OF CONDUCT](https://github.com/kunjgit/GameZone/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I have followed proper naming convention showed in [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md)
- [x] I have added screenshot for website preview in assets/images 
- [x] I have added entries for my game in main README.md
- [x] I have added README.md in my folder 
- [x] I have specified the respective issue number for which I have requested the new game.

<hr>

## Add your screenshots(Optional) 📸
Eco_Warrior

--- 
<br>

## Thank you soo much for contributing to our repository 💗

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,7 +110,7 @@ This repository also provides one such platforms where contributers come over an
...
-| [Tilting Maze](https://github.com/kunjgit/GameZone/tree/main/Games/Tilting_Maze)                                 | [Simon Game Challenge](https://github.com/kunjgit/GameZone/tree/main/Games/Simon_Game_Challenge)        | [Snake Game](https://github.com/kunjgit/GameZone/tree/main/Games/Snake_Game)                                 | [Dino Runner Game](https://github.com/kunjgit/GameZone/tree/main/Games/Dino_Runner_Game)                           |
+| [Eco Warrior](https://github.com/Antiquely3059/GameZone/tree/main/Games/Eco%20Warrior)                                 | [Tilting Maze](https://github.com/kunjgit/GameZone/tree/main/Games/Tilting_Maze)                                 | [Simon Game Challenge](https://github.com/kunjgit/GameZone/tree/main/Games/Simon_Game_Challenge)        | [Snake Game](https://github.com/kunjgit/GameZone/tree/main/Games/Snake_Game)                                 | [Dino Runner Game](https://github.com/kunjgit/GameZone/tree/main/Games/Dino_Runner_Game)                           |
```
Comment (ishita-43) [inline README.md:113]: change the link text as other specify in the pattern only placing your game name in the end!!

**Review by ishita-43 — CHANGES_REQUESTED** [review]
Hoping you will make desired changes soon!

**Comment by Antiquely3059** [comment]
Updated the link text as you requested.

**Comment by Antiquely3059** [comment]
Please review it

**Review by ishita-43 — CHANGES_REQUESTED** [review]

**Comment by Antiquely3059** [comment]
I have updated the changes as you requested.
Please review it.

File: README.md [hunk 2]
```diff
@@ -796,6 +796,20 @@ This repository also provides one such platforms where contributers come over an
 
 
 
+
+=======
```
Comment (ishita-43) [inline README.md:800]: remove this line!

**Review by ishita-43 — CHANGES_REQUESTED** [review]

**Comment by Antiquely3059** [comment]
I have updated the changes as you requested.
Please review it.

**Review by ishita-43 — APPROVED** [review]
Good to go!!

**Comment by ishita-43** [comment]
also Resolve merge conflicts

**Comment by Antiquely3059** [comment]
Resolved it

File: README.md
Comment (ishita-43) [inline README.md:1]: make sure  you added your game entry in this file and also that merge conflicts are resolved

**Comment by kunjgit** [comment]
Thank you  @Antiquely3059 , for your valuable time and contribution in our GameZone 💗. 
 It’s our GameZone, so Let’s build this GameZone altogether !!🤝
  Hoping to see you soon with another PR again 😇 
 Wishing you all the best for your journey into Open Source🚀

**Pull request closed without merging**